README.md

```diff
@@ -1,2 +1,7 @@
 # eap-cloud-galleon-pack
 Galleon feature-pack for EAP on the Cloud
+
+The ``org.jboss.eap.cloud:eap-cloud-galleon-pack`` Galleon feature-pack provisions a set of additional features allowing you to configure an \
+EAP server to work on the cloud. 
+
+[Detailed documentation](doc/index.md) of the EAP cloud Galleon feature-pack.
```

Comment:
That's a mouthful but we should use the same terminology to be consistent "Galleon feature-pack for EAP on the Cloud" (or say we will refer to it as the "EAP cloud feature pack"?)

Reply:
Fixed.